app/services/ingest_iati_activities.rb

```diff
@@ -159,7 +159,7 @@ def call
         parent_activity: new_activity
       )
 
-      budget.save!
+      budget.save!(validate: false)
```

Comment:
Can we flag budgets (or any entity we ingest) as ingested (or something like `source: ingested`) like we propose on Activity and then a simple validation context would work well here, something like:

`validates :period_start_date, :period_end_date, date_within_boundaries: true, unless: :ingested?`

Reply:
Bah! That's actually a conditional vlidation - soz, but I still think the approach is sound!